### ISSUE TITLE
81-column wide output

### ISSUE THREAD

**Issue by bretmartin** [ansible/ansible#11883]
I suppose I am old-fashioned for using 80-column terminals in many places still, but I noticed this change in Ansible recently. Was it intentional?

On e16b24c01e:

```
PLAY ****************************************************************************
[...]
```

On tag `v1.9.2-1`:

```
PLAY [localhost] **************************************************************
```

The output used to be 79c wide but now it's 81c wide.

I realize this may seem like a trivial aesthetic issue -- I'm hoping it was a minor, unintentional change.

**Comment by bcoca** [comment]
i actually want to change it to be adjustable to terminal size, like i did for ansible-doc

**Comment by bcoca** [comment]
for what it is worth, the code currently looks like:

```
    wrapped = textwrap.wrap(new_msg, 79)
```

**Comment by jimi-c** [comment]
This was fixed in 5f0359c1.

Thanks!
